.github/workflows/javadoc.yml

```diff
@@ -3,21 +3,28 @@ name: JavaDoc
 on:
   push:
     branches:
-      - 2.2.X
+      - 4.0.X
   workflow_dispatch:
 
 jobs:
   javadoc:
     runs-on: ubuntu-latest
+    
+    env:
+      JDK_VERSION: 17
+    
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Echo Java Version
+        run: >
+            java -version
```

Comment:
this seems like debugging attempt

Reply:
Yes. I just wanted it to replay the java version to varify that it is using the correct version.  It is harmless, I'd prefer if we leave it there, because modifying these workflows is a PITA. :)
(I'm sitting in the Air France Lounge at the Airport)

Reply:
Connected with @AlexanderSaydakov briefly on slack and we're both ok leaving it. Definitely debugging (I had the same reaction) but harmless and potentially useful in the future.